examples/testing/datasets/imagenet2012/5.0.0/image.image.json

```diff
@@ -0,0 +1 @@
+{"encoding_format": "jpeg", "shape": [-1, -1, 3]}
```

Comment:
Have you tried to see if we can avoid this by loading these files directly via

```
import tfds
tfds.__path__
```

Reply:
Yup, I gave that a shot. testing/metadata not exposed as part of the library in the python packages sadly. 

I spoke with the TFDS team, they recommended to copy the metadata files for now but a better solution is WIP (ETA: later this year).